libs/journeys/ui/src/components/Card/OverlayContent/OverlayContent.tsx

```diff
@@ -16,6 +19,16 @@ export function OverlayContent({
   hasFullscreenVideo = false
 }: OverlayContentProps): ReactElement {
   const { variant } = useJourney()
+  const cardOverlayRef = useRef<any>()
+  const trigger = useScrollTrigger({
+    disableHysteresis: true,
+    target: cardOverlayRef.current,
+    threshold: 20
+  })
+  const isScrollable = (): boolean => {
+    const box = cardOverlayRef.current
+    return box != null ? box.scrollHeight > box.clientHeight : false
+  }
   const enableVerticalScroll: SxProps = {
     overflowY: 'scroll',
```

Comment:
```suggestion
  const cardOverlayRef = useRef<any>()
  
  const trigger = useScrollTrigger({
    disableHysteresis: true,
    target: cardOverlayRef.current,
    threshold: 20
  })
  
  const isScrollable = (): boolean => {
    const box = cardOverlayRef.current
    return box != null ? box.scrollHeight > box.clientHeight : false
  }
  
  const enableVerticalScroll: SxProps = {
    overflowY: 'scroll',
```

Add spacing to break up the code blocks for better reading